keyboards/dumbo/rev1/rev1.h

```diff
@@ -0,0 +1,43 @@
+/* Copyright 2020 Adam Naldal <adamnaldal@gmail.com>
```

Comment:
Do you plan on having further revisions of the board?  

If not, it may be worth/preferable to squash this so there is no revision folder, to keep things simpler.

Reply:
Good point. 
I will remove it tonight, as i am not planning on making any major changes to the board in the near future.

I am kind of new to the pull request system, is there anything i need to do after i push the changes to my qmk-fork?

Reply:
Nope, just commit to the same branch that has these change, and push.

Reply:
Allright i have removed the rev1 folder and tested that everything still works :)